pkg/manager-server/file_system_lvm.go

```diff
@@ -190,9 +191,16 @@ func (f *FileSystemLvm) Mount(mountpoint string) error {
 		return err
 	}
 
-	if _, err := f.run(fmt.Sprintf("mount --bind %s %s", f.devPath(), mountpoint)); err != nil {
+	mounted, err := f.IsMounted(mountpoint)
```

Comment:
Similarly here, should you check whether the directory exists on line 186 before creating it?